backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -59,8 +67,162 @@ def test_start_project(self):
 # class log_card_review_request_cancelled_Tests(TestCase):
 
 
-# class log_card_moved_to_complete_Tests(TestCase):
+class log_card_moved_to_complete_Tests(TestCase):
+
+    def test_card_to_complete_creates_one_log_entry(self):
+
+        actor_user = UserFactory(is_superuser=True)
+        card = factories.AgileCardFactory(
+            status=AgileCard.COMPLETE,
+            content_item=factories.ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            ),
+        )
+        card.save()
+
+        creators.log_card_moved_to_complete(card, actor_user)
```

Comment:
This test is in the wrong file.

You just envoked the function. You called it.

This file is all about making sure things are called when they are supposed to be called. This file tests that things are envoked when they need to be. 

Please move the code.

Or please choose a new and meaningful standard and then make all the existing action creator tests fit that standard.